package.json

```diff
@@ -186,16 +186,14 @@
     "multiformats": "^11.0.0",
     "node-forge": "^1.1.0",
     "protons-runtime": "^4.0.1",
+    "uint8arraylist": "^2.4.3",
     "uint8arrays": "^4.0.2"
   },
   "devDependencies": {
     "@types/mocha": "^10.0.0",
-    "aegir": "^37.0.12",
+    "aegir": "^38.1.2",
     "benchmark": "^2.1.4",
-    "protons": "^6.0.0",
```

Comment:
`protons` is used in the `generate` npm script.

Please update the `dep-check` command to ignore protons - `"dep-check": "aegir dep-check -i protons"`

Reply:
Are there any other packages that `dep-check` might ignore? May be worthwhile to revisit the functionality of `dep-check` if this isn't the only exception.

Reply:
It only tests for modules used in code, it doesn't include npm scripts.

I suppose you could get aegir to look at the `bin` object of every dependency and see if any of the commands are used in the npm scripts of the current project and add the containing module to the ignore list if so but it's a lot of work and may end up with some false positives if two deps export the same command.

Probably not worth the effort IMO.